api/v1alpha2/virtualmachine_network_types.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // VirtualMachineNetworkRouteSpec defines a static route for a guest.
 type VirtualMachineNetworkRouteSpec struct {
-	// To is an IP4 or IP6 address.
+	// To is either "default", or an IP4 or IP6 address.
```

Comment:
Can we use an empty value as the default instead, so users don't have to explicitly set "default" here? IOW, webhook only calls `net.ParseCIDR(r.To)` if the field is not empty.

Reply:
I think that makes the output look too incomplete and confusing to see just a `via` field